lib/msal-browser/docs/access-token-proof-of-possession.md

```diff
@@ -151,6 +151,10 @@ The Proof-of-Possession authentication scheme relies on an asymmetric cryptograp
 
 In the event of refreshing a bound access token, MSAL will delete the cryptographic keypair that was generated when requesting the expired bound access token, generate a new cryptographic keypair for the new access token, and store the new keypair in the keystore.
 
+## Advanced feature: User generated cryptographic keypair
+
+MSAL does not sign the POP tokens if the application chooses to generate its own random cryptographic keypair and returns the unsigned `access_token` generated by the Service to the application. This is fairly advanced feature and we recommend not to use it unless the developer understands POP and finds a use case on their end to manage the `reqCnf` generation themselves. `reqCnf` can be provided as an optional parameter on the `request` type.
```

Comment:
I would also add a clear usage example below and make note of the fact that you should not provide the other PoP params.

```suggestion
> :warning: This is an advanced feature and we do not recommend its use unless you are intimately familiar with the Proof of Possession protocol and have a requirement to manage your own keypairs. For most use cases we recommend implementing PoP as described in the rest of this document.

If you would like to generate your own cryptographic keypair and be responsible for signing the resulting tokens you can provide your own `reqCnf` value on the request. If this value is provided MSAL will ensure the token issuer embeds it in the token but will return the token _unsigned_. It is then the application's responsibility to sign the token before attempting to send it to the intended resource. 
```

Reply:
Updated text